doc/design/related-operator-framework-projects.md

```diff
@@ -0,0 +1,60 @@
+# Related Operator Framework Projects
+
+There are a number of projects that enable OLM to install and manage the lifecycle of an operator. Each of these projects have specific goals and their own GitHub repository. Due to the distributed nature of these projects, it can be difficult to know where a change must be made when introducing a new feature or bug fix to OLM.
+
+It is the goal of this document to keep track of the various projects enable OLM to install and manage the lifecycle of operators.
```

Comment:
```suggestion
It is the goal of this document to keep track of the various projects which enable OLM to install and manage the lifecycle of operators.
```

Reply:
The suggested change has been accepted in the latest version of the pull request